src/gh-repos.ts

```diff
@@ -280,11 +280,13 @@ export default class GitHubRepositoriesProvider {
 						optional: false,
 					};
 					param.name = property.name as string;
-					param.description = property.description as string;
+					const descripArray = property.description.split("=>");
+					param.description = descripArray[0] as string;
 					param.optional = property.optional as boolean;
 					param.type = yoTypeRegex.test(property.type.names.find((name: string) => name.includes("yo")))
 						? yoTypeRegex.exec(property.type.names.find((name: string) => name.includes("yo")))[0]
 						: "input";
+					param["env"] = descripArray?.[1]?.replace(/(.*)env:/, "") as string;
```

Comment:
Do you need some kind of error handling in here?
Is it possible there is no description to split or no `=>` value.
Especially because the optional concatenation only applies to undefined.
If an empty string is processed with the function `split`, only an array with one value is returned.

Do you want to return `undefined` here or a empty string if value is not found.
So empty string would be:
```suggestion
					param["env"] = descripArray?.[1]?.replace(/(.*)env:/, "") || "" as string;
```